src/CADL.Extension/Emitter.Csharp/src/type/OperationFinalStateVia.ts

```diff
@@ -4,5 +4,7 @@
 export enum OperationFinalStateVia {
     AzureAsyncOperation,
     Location,
-    OriginalUri
+    OriginalUri,
+    OperationLocation,
+    LocationOverride,
```

Comment:
What does 'LocationOverride' mean?  Do we have any spec to describe it?

Reply:
Do we have test cases to cover all above?

Reply:
> What does 'LocationOverride' mean? Do we have any spec to describe it?

It's the equivalence of https://github.com/Azure/autorest.csharp/blob/23d44f529c1e7bb7005aac9516f532713e50bf1a/src/assets/Generator.Shared/OperationFinalStateVia.cs#L8-L15



> Do we have test cases to cover all above?

We don't have all test cases. I think right now it's OK, since we only emit `Location` and `Operation-Location` which are covered by test cases. Here I just want to make sure typescript codes are aligned with csharp codes.